packages/twenty-server/src/engine/api/rest/api-rest-query-builder/factories/input-factories/filter-utils/check-filter-enum-values.ts

```diff
@@ -0,0 +1,42 @@
+import { BadRequestException } from '@nestjs/common';
+
+import { ObjectMetadataInterface } from 'src/engine/metadata-modules/field-metadata/interfaces/object-metadata.interface';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+
+export const checkFilterEnumValues = (
```

Comment:
check-filter-enum-values-or-throw if we throw. Do we actually want to throw here? we could return a boolean instead and leave it to the code calling the util to throw. I always feel it's not the util responsibiltiy to throw as depending on the context you only want to log (which forces a try catch) or to throw